README.md

```diff
@@ -112,6 +118,13 @@ python makeFastInput.py -d /store/user/lpcsusyhad/SusyRA2Analysis2015/Run2Produc
 python makeFastInput.py -d /store/user/lpcsusyhad/SusyRA2Analysis2015/Run2ProductionV18/scan/ -m MC2018 -y MC2018HEM -s input/input_sets_skim_fast_2018HEM.txt -c input/input_sets_DC_fast_2018HEM.txt -e batch/exportSkimFast2018HEM.sh
 ```
 
+If you are trying to make input files for the fullsim signal scans, then the commands become:
+```
+python makeFastInput.py -f -d /store/user/lpcsusyhad/SusyRA2Analysis2015/Run2ProductionV18/scan/ -m MC2016 -s input/input_sets_skim_signalscan_2016.txt -c input/input_sets_DC_signalscan_2016.txt -e batch/exportSkimSignalScan2016.sh -k T1bbbb,T1qqqq,T1tttt,T2bb,T2qq,T2tt,T5qqqqVV,TChiHH
```

Comment:
update these commands with `-x 2`?

Reply:
Done, but with `-x 1`, which is the appropriate number for fullsim.